addons/test_mail/tests/test_performance.py

```diff
@@ -303,7 +303,7 @@ def test_activity_full(self):
             # voip module read activity_type during create leading to one less query in enterprise on action_feedback
             _category = activity.activity_type_id.category
 
-        with self.assertQueryCount(admin=14, employee=13):  # tm: 10 / 10
+        with self.assertQueryCount(admin=18, employee=17):  # tm: 10 / 10
```

Comment:
This indicates you are adding a lot of queries for few real reasons. Current behavior already sends a lot of notifications, not sure we should continue adding them and degrading performances. Somehow only some specific flows probably really require to send bus notifications.